guide/additional-info/changes-in-v12.md

```diff
@@ -1833,11 +1833,11 @@ All the `.send***()` methods have been removed in favor of one general `.send()`
 
 #### User#block
 
-`user.block()` has been removed entirely, along with all other user account-only methods.
+`user.block()` was removed entirely, along with all other user account-only methods.
 
 #### User#displayAvatarURL
 
-`user.displayAvatarURL` is now a method, as opposed to a property. It also allows you to determine the file format and size to return. If the `dynamic` option is provided you will receive a `.gif` URL if the image is animated, otherwise it will fall back to the specified `format` or its default `.webp` if none is provided.
+`user.displayAvatarURL` is now a method, as opposed to a property. It also allows you to determine the file format and size to return. If the `dynamic` option is provided, you will receive a `.gif` URL if the image is animated; otherwise, it will fall back to the specified `format` or its default `.webp` if none is provided.
```

Comment:
```suggestion
`user.displayAvatarURL` is now a method, as opposed to a property. It also allows you to determine the file format and size to return. If you provide the `dynamic` option, you will receive a `.gif` URL if the image is animated; otherwise, it will fall back to the specified `format` or its default `.webp`.
```